### ISSUE TITLE
Support multiple upload queues in one form

### ISSUE THREAD

**Issue by Jako** [Jako/AjaxUpload#8]

**Comment by sonicpunk** [comment]
What is the roadmap on this??

**Comment by Jako** [comment]
Too less time at the moment.

**Comment by jcdm** [comment]
@Jako A far as you're aware, would this be as simple as making the snippets able to process comma delimited lists of settings?

i.e. if someone followed the path through the preHooks and Hooks looking for instances of ajaxuploadFieldname, ajaxuploadTarget and ajaxuploadUid and made them able to run per-setting in a comma separated list, would that likely do the job or are there reasons to believe it would be harder than that?

**Comment by jcdm** [comment]
I've just attempted to place two different upload queues on the same page and it appears to have worked*.

* when I say it's worked, I'm not saving the resulting file name to the FormItSaveForm database save so I can't verify that the two queues don't get confused at that level but I have a hook running that is able to use the SESSION data and find the respective queues. ALl files are sitting in the one upload path and I'm able to move them to their respective target destinations respectively using a custom written hook.

**Comment by Jako** [comment]
@jcdm If you want, you can send the stripped custom hook and I will add this to the docs. Or you can create a PR here: https://github.com/Jako/AjaxUpload/tree/master/docs

**Comment by Jako** [comment]
No, @jcdm did not sent his solution. But there is a difference between multiple queues on one page (he mentioned this) and in one form handled by FormIt.

The upload data is saved in the session, the upload queues are separated with uids. But it is not possible to have multiple uids in the AjaxUpload hooks. The hook code has to be extended to use multiple fieldnames and multiple uids. Maybe there has to be something added in the main class too.

**Comment by Jako** [comment]
Multiple upload queues are available with AjaxUpload 2.0.0.
